docs/tutorials/multimodal/advanced_topics/focal_loss.md

```diff
@@ -0,0 +1,53 @@
+# How to use FocalLoss
+:label:`sec_automm_focal_loss`
+
+In this tutorial, we introduce how to use `FocalLoss` from the AutoMM package for balanced training.
+FocalLoss is first introduced in this [Paper](https://www.microsoft.com/en-us/research/uploads/prod/2021/10/model-size-graph.jpg)
+and can be used for balancing hard/easy samples as well as un-even sample distribution among classes. This tutorial demonstrates how to use `FocalLoss`.
+
+## Create Dataset
+We use the shopee dataset for demonstration in this tutorial.
+```{.python .input}
+from autogluon.multimodal.utils.misc import shopee_dataset
+
+download_dir = "./ag_automm_tutorial_imgcls_focalloss"
+train_data, test_data = shopee_dataset(download_dir)
+```
+
+## Create and train `MultiModalPredictor` with `FocalLoss`
+We specify the model to use `FocalLoss` by setting the `"optimization.loss_function"` to `"focal_loss"`.
+There are also three other optional parameters you can set.
+
+`optimization.focal_loss.alpha` - a list of floats which is the per-class loss weight that can be used to balance un-even sample distribution across classes.
```

Comment:
We need to describe a bit more on how to compute alpha, otherwise people don't know how to set the values. I think the easiest way is to use the inverse of number of samples in each class.